### PR TITLE
Update Readme.md to reference omi ntlm setup doc.

### PR DESCRIPTION
**Pull request by yakman2020**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,21 +123,31 @@ ip -f inet addr show dev eth0
 Connecting from Linux to Windows
 --------------------------------
 
-*NOTE: This is pre-release and does have plenty of known problems. We are pushing it to master so you can try it out.*
-*Please don't file issues on crashes or hangs yet as it is probably already known about.*
+For Centos 7.3 and later, and Ubunutu 16.04 or later, Powershell supports Secure Protected Negotiated authentication (SPNEGO).
```
Comment (niroyb) [inline README.md]: *Ubuntu

File: README.md [hunk 2]
```diff
@@ -123,21 +123,31 @@ ip -f inet addr show dev eth0
 Connecting from Linux to Windows
 --------------------------------
 
-*NOTE: This is pre-release and does have plenty of known problems. We are pushing it to master so you can try it out.*
-*Please don't file issues on crashes or hangs yet as it is probably already known about.*
+For Centos 7.3 and later, and Ubunutu 16.04 or later, Powershell supports Secure Protected Negotiated authentication (SPNEGO). 
+This allows the use of NTLM based authentication and encryption of traffic over the http connection.  Use of SPNEGO authentication is
+more secure than basic authentication on http, and less complex than https.  Support of NTLM authentication on MacOS is deprecated by apple. 
+We are currently working on Kerberos protocol support which will allow SPNEGO or direct Kerberos authentication on MacOS as well as at least some
+older linux distributions.
```
Comment (niroyb) [inline README.md]: *Linux

File: README.md [hunk 3]
```diff
@@ -123,21 +123,31 @@ ip -f inet addr show dev eth0
...
 
-WinRM server needs to be configured to allow unencrypted traffic and accept basic authentication for inbound connections.
-*NOTE: This sends passwords over unencrypted http.*
+In order to perform NTLM authentication there must be matching credentials on both ends of the transaction. iThe necessary setup of the ntlm credentials
```
Comment (niroyb) [inline README.md]: *The

File: README.md [hunk 4]
```diff
@@ -123,21 +123,31 @@ ip -f inet addr show dev eth0
...
+for both server and client is decribed in the document [setup-ntlm-omi]( https://github.com/Microsoft/omi/doc/setup-ntlm-omi.md).
```
Comment (niroyb) [inline README.md]: *described

File: README.md (hunk 3 above)
Comment (paulcallen) [inline README.md]: Still have typo

File: README.md [hunk 5]
```diff
@@ -123,21 +123,31 @@ ip -f inet addr show dev eth0
...
+We are currently working on Kerberos protocol support which will allow SPNEGO or direct Kerberos authentication on MacOS as well as at least some
+older linux distributions.
 
-WinRM server needs to be configured to allow unencrypted traffic and accept basic authentication for inbound connections.
-*NOTE: This sends passwords over unencrypted http.*
+In order to perform NTLM authentication there must be matching credentials on both ends of the transaction. iThe necessary setup of the ntlm credentials
+for both server and client is decribed in the document [setup-ntlm-omi]( https://github.com/Microsoft/omi/doc/setup-ntlm-omi.md). 
 
-We are working on SPNEGO authentication with encryption over HTTP and this will hopefully be in place soon.
+If you are using not using SPNEGO authentication, or wish to use basic authentication on https, the WinRM server needs to be configured to allow unencrypted traffic
```
Comment (paulcallen) [inline README.md]: Are you using or not using, I am confused!

**Pull request merged**